src/DynamoApplications/StartupUtils.cs

```diff
@@ -378,7 +465,7 @@ private static bool PreloadASM(string asmPath, out string geometryFactoryPath, o
                 CLIMode = CLImode
             };
             config.AuthProvider = CLImode ? null : new Core.IDSDKManager();
-            config.UpdateManager = CLImode ? null : InitializeUpdateManager();
+            config.UpdateManager = CLImode ? null : OSHelper.IsWindows() ? InitializeUpdateManager() : null;
```

Comment:
what is windows only about updatemanager?

Reply:
Hopefully we can get rid of most of the Windows only APIs from DynamoCore
This can be done in steps